Sources/Sentry/SentrySDK.m

```diff
@@ -431,6 +432,32 @@ + (void)captureFeedback:(SentryFeedback *)feedback
     [SentrySDK.currentHub captureFeedback:feedback];
 }
 
+#if TARGET_OS_IOS && SENTRY_HAS_UIKIT
+
++ (void)showFeedbackWidget
```

Comment:
`m`: We could encapsulate the feedback methods into a `SentryFeedbackAPI` similar to `SentryReplayAPI` so it can be used like this:

```
SentrySDK.feedback.showWidget()
```

Reply:
While I dislike having to instantiate a singleton to do this and use instance methods, I do like the organizational aspect of it.